derrida/books/views.py

```diff
@@ -709,12 +710,13 @@ def get(self, request, *args, **kwargs):
                 # need to be made relative to current url
                 local_response[header] = value
 
-        # special case, for deep zoom (hack)
-        if kwargs['mode'] == 'info':
-            data = remote_response.json()
+        # special case, for deep zoom json info response
+        if kwargs['mode'] == 'info' or (kwargs['mode'] == 'iiif' and kwargs['url'] in ['/', '/info.json']):
```

Comment:
http://127.0.0.1:8000/library/jakobson-fundamentals-of-language-1963/gallery/images/p-116/iiif

Would resolve and show the PUL version. This may be overkill, but we could add one more option for the if statement

```suggestion
        if kwargs['mode'] == 'info' or (kwargs['mode'] == 'iiif' and kwargs['url'] in ['/', '/info.json', '']):
```